src/Features/Core/Portable/UseThrowExpression/AbstractUseThrowExpressionDiagnosticAnalyzer.cs

```diff
@@ -62,7 +62,7 @@ protected override void InitializeWorker(AnalysisContext context)
                 s_registerOperationActionInfo.Invoke(startContext, new object[]
                 {
                     new Action<OperationAnalysisContext>(operationContext => AnalyzeOperation(operationContext, expressionTypeOpt)),
-                    ImmutableArray.Create(OperationKind.ThrowStatement)
+                    ImmutableArray.Create(OperationKind.ExpressionStatement)
```

Comment:
My thought here was that invoking `Parent` on the ThrowExpression might be more expensive than checking to see if the child is a ThrowExpression.